graphql/tests/query.rs

```diff
@@ -586,6 +592,32 @@ fn query_variables_are_used() {
     );
 }
 
+#[test]
+fn bytes_child() {
```

Comment:
Do we already have a test for when the parent is `Bytes` and the child is `ID`?

Reply:
Yes, for example `can_query_one_to_many_relationships_in_both_directions` does that with `{writtenSongs .. { writtenBy { id } }` when songs use a `Bytes` id

Reply:
That's not quite what we want (`writtenBy` is derived) Just added a test that checks that combination